wannierberri/__tabulate.py

```diff
@@ -384,9 +394,12 @@ def plot_path_fat(
 
         if save_file is None:
             plt.show()
+            plt.close()
+        elif save_file == "":
+            pass
```

Comment:
Is this option to make the figure not closed? If so, what about adding `close_fig` argument?

Reply:
done, also added several other options